erpnext/hr/doctype/employee_onboarding/test_employee_onboarding.py

```diff
@@ -39,7 +39,7 @@ def test_employee_onboarding_incomplete_task(self):
 
 		# complete the task
 		project = frappe.get_doc('Project', onboarding.project)
-		for task in frappe.get_all('Task', dict(project=project.name)):
+		for task in frappe.get_all('Task', dict(default_project=project.name)):
```

Comment:
Have you fixed the test records as well?

Reply:
Hey @neilLasrado Line number 13 which Create Two Task for it. it Has a controller EmployeeBoardingController which creates a task on submitting employee onboarding.
if you look for file erpnext/hr/utils.py line no 58 which append project to task. so that is fixed